packages/offix-datastore/tests/DataStore.test.ts

```diff
@@ -22,11 +19,32 @@ function getIndexedDB() {
     });
 }
 
+interface Note {
+    id?: string;
+    title: string;
+    description: string;
+}
+
+interface Comment {
+    id?: string;
+    title: string;
+    noteId: string;
+}
+
+let NoteModel: Model<Note>;
+
 beforeEach(() => {
-    configure([
-        { __typename: "Note" },
-        { __typename: "Comment" }
-    ]);
+    NoteModel = model<Note>("user_Note", {
```

Comment:
I will need to think about this format. 
We need to specify:
1) Type
2) Database key
3) Format (which might mean many many fields


This is too ugly and too much boilerplate to become user API.
Maybe we can actually generate this and wrap it into NoteModel classes that will accept database keys (this way you can have the same type existing in database twice - with different filters.

Reply:
> * Type
> * Database key
> * Format (which might mean many many fields

I made some changes to accommodate options like these ones. I don't know what you mean by "Format", so its a "TODO" right now.

> This is too ugly and too much boilerplate to become user API.
 
Yeah. We will generate it.

> this way you can have the same type existing in database twice - with different filters.

Not sure what you mean by this.
@wtrocki

Reply:
@wtrocki